models/BaseAdobe.cfc

```diff
@@ -697,6 +697,7 @@ component accessors=true extends='cfconfig-services.models.BaseConfig' {
 		if( !isNull( thisConfig.keyAlias ) ) { setMailSignKeyAlias( thisConfig.keyAlias ); }
 		if( !isNull( thisConfig.keypassword ) ) { setMailSignKeyPassword( passwordManager.decryptMailServer( thisConfig.keypassword ) ); }
 		if( !isNull( thisConfig.mailsentloggingenable ) ) { setMailLogEnabled( thisConfig.mailsentloggingenable ); }
+		if( !isNull( thisConfig.maintainConnections ) ) { setMailLogEnabled( thisConfig.maintainConnections ); }
```

Comment:
You're setting this into the MailLogEnabled setting, which is incorrect. A new setting needs created in `CaseConfig.cfc` to store this.  I would call it `mailMaintainConnections`.

Reply:
followed your suggestion here - I wasn't paying enough attention on the initial pull request